readme.markdown

```diff
@@ -447,6 +447,21 @@ $ node object.js test/x.js test/y.js
 {"type":"end","test":2}
 ```
 
+An convenient alternative to achieve the same:
+```js
+// report.js
+var test = require('tape');
+var path = require('path');
```

Comment:
path variable is unused.

Reply:
You're absolutely right. I fixed this.
Thanks for your comment.